source/encryption.rst

```diff
@@ -0,0 +1,111 @@
+Data at rest encryption
+************************
+
+`Ful data at rest encryption in Percona XtraDB Cluster <https://www.percona.com/doc/percona-xtradb-cluster/LATEST/management/data_at_rest_encryption.html>`_ is supported by the Operator since version 1.4.0.
+
+..note:: "Data at rest" means inactive data stored as files, database records, etc.
+
+To implement these features, the Operator uses ``keyring_vault`` plugin,
+which ships with Percona XtraDB Cluster 8.0, and utilizes `HashiCorp Vault <https://www.vaultproject.io/>`_ authentication server. 
+
+.. contents:: :local:
+
+.. _install-vault:
+
+Installing and configuring Vault
+-------------------------------
+
+The following steps will deploy Vault on Kubernetes with the `Helm 3 package manager <https://helm.sh/>`_:
```

Comment:
Maybe it is needed to mention that user feel free to setup Vault as he wants, steps 1-4 are only illustrative